src/validation/fields/public-register.validation.ts

```diff
@@ -11,3 +11,11 @@ export const public_register_validations = [
     .custom((value, { req }) => checkFieldIfRadioButtonSelected(req.body.is_on_register_in_country_formed_in === '1', ErrorMessages.PUBLIC_REGISTER_NUMBER, value) )
     .custom((value, { req }) => checkMaxFieldIfRadioButtonSelected(req.body.is_on_register_in_country_formed_in === '1', ErrorMessages.MAX_PUBLIC_REGISTER_NUMBER_LENGTH, 32, value) )
 ];
+
+// To be removed, checking only MAX for BOs fields
```

Comment:
i would remove the //to be removed comment, as someone may delete it!  just have // checking only MAX for BOs fields